openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -168,6 +168,30 @@ <h1 translate>Motions</h1>
               </li>
             </ul>
           </span>
+          <!-- Comment filter -->
+          <span uib-dropdown ng-if="commentsFieldsNoSpecialComments.length > 0">
```

Comment:
Please show the comments filter only for users with permission  (can_see_and_manage_comments). - Otherwise we have to hide all private comments from filter... I think it is enough to show this filter only for managers who also can edit these fields.